src/views/players/player/player_troll.cpp

```diff
@@ -17,12 +17,15 @@ namespace big
 			components::player_command_button<"rcplayer">(g_player_service->get_selected());
 
 			static int bounty_value = 0;
-
+			static bool anonymous = true;
```

Comment:
Would be nice to save this.

Reply:
You can look at how `breakupcheating` is implemented, it does the same for the breakup message.
https://github.com/YimMenu/YimMenu/blob/081dd6d69418c7b79c1c6ad812bf389c0d872e1f/src/backend/commands/player/kick/breakup_kick.cpp#L65